packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -166,6 +176,14 @@ async function applyCategoryTemplate(category, template_lines, month, force) {
           template.from = format(spend_from, 'yyyy-MM');
         }
         break;
+      case 'schedule':
+        if (!all_schedule_names.includes(template.name)) {
+          console.log(
+            `${category.name}: Schedule ${template.name} does not exist`,
+          );
```

Comment:
Can you add this to the `errors` array like other problems are reported in this function?

Reply:
Oops, I missed that when I merged in the new error-reporting.